python/lsst/ts/MTAOS/model.py

```diff
@@ -217,6 +220,9 @@ def __init__(
 
         self._user_gain = self.ofc.default_gain
 
+        self.wep_process = None
+        self.wep_process_started = make_done_future()
```

Comment:
Please help to document the `wep_process_started` in the doc string of `Attributes`. In addition, I suggest to rename this to be something like the `task_run_wep_process` because the original name suggests the value of `wep_process_started ` to be the `bool` instead of `asyncio.Task`. Thanks!